apps/docs/components/blocks/Gallery.md

```diff
@@ -15,6 +15,10 @@ The Gallery presents a visually appealing and user-friendly collection of images
 
 The Gallery is primarily intended for use on product pages, where it serves as an effective tool for showcasing product images. The choice between the vertical and horizontal variants should depend on the layout of the product page, ensuring optimal visual presentation. Additionally, the variant with bullets should be considered, particularly for mobile devices, as it offers a compact and easily accessible navigation format.
 
+## Accessibility notes
+
+The Gallery supports the use of the keyboard (Tab/alt+Tab) to navigate through images.
```

Comment:
```suggestion
The Gallery supports the use of the keyboard (Tab/shift+Tab) to navigate through images.
```
I think shift + tab should be not alt + tab

Reply:
dbe479c657cff5f191c4cf57ed29a3af3498eeab